tests/unit/test_loaders.py

```diff
@@ -332,7 +332,7 @@ def test_AgeRange_different_ranges(range_a, range_b):
 
     assert hash(a) != hash(b)
     assert a != b
-    assert not (a == b)
+    assert not a == b  # pylint: disable=unneeded-not
```

Comment:
Why do we need both these lines? Maybe I'm being daft but they superficially look like they're doing the same thing. Is it to do with how `=` and `!=` are defined for an `AgeRange`?

Reply:
Maybe we could call `a.__eq__(b)` to be more specific. `!=` and `==` call different methods.

Reply:
I think leave as is. I was just curious.